.github/workflows/smoke-test.yaml

```diff
@@ -70,6 +71,40 @@ jobs:
           kubectl apply -f ${{ env.CONFORMANCE_TEMPLATE }}
           kubectl wait --for=condition=Available --all deployment --timeout=5m
 
+      - name: Dump cilium related logs and events
+        if: ${{ failure() }}
```

Comment:
I only dump the logs in case of _failure_ only.

Testing was done with _always()_  here https://github.com/cilium/cilium/pull/12286/checks?check_run_id=813681586